test/base/DummyDAppControl.sol

```diff
@@ -8,11 +8,23 @@ import "../../src/contracts/types/UserCallTypes.sol";
 import "../../src/contracts/types/SolverCallTypes.sol";
 
 contract DummyDAppControl is DAppControl {
-    constructor(address escrow)
+    constructor(address escrow, address governance, CallConfig memory _callConfig)
         DAppControl(
             escrow,
-            address(0),
-            CallConfig(
+            governance,
+            _callConfig
+        )
+    { }
+
+    function _preOpsCall(UserOperation calldata) internal virtual override returns (bytes memory) { }
+    function _allocateValueCall(address, uint256, bytes calldata) internal virtual override { }
+    function getBidFormat(UserOperation calldata) public view virtual override returns (address) { }
+    function getBidValue(SolverOperation calldata) public view virtual override returns (uint256) { }
+}
+
+library CallConfigBuilder {
+    function allFalseCallConfig() internal pure returns (CallConfig memory) {
+        return CallConfig(
                 false,
                 false,
                 false,
```

Comment:
I just realized that we can remove that long list of `false` flags with:
```solidity
function allFalseCallConfig() internal pure returns (CallConfig memory) { }
```
It returns the same.